spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  let(:user) do
+    User.create(
+      name: 'Benson',
+      photo: 'https://www.google.com/imghp?hl=en&ogbl',
+      bio: 'Full Stack Developer',
+      posts_counter: 2
```

Comment:
- [ ] Your tests are failing due to a mismatch between the attribute names in your schema and your code. You have used `posts_counter` in your test files, models, and views, but in your schema it is `post_counter`. This causes an unknown attribute `posts_counter` error when running the tests. To fix this, you need to either change the attribute name in your schema to posts_counter or update your code to use post_counter consistently. This will make your tests pass.

Here is a screen shot of your failing tests:

<img width="784" alt="Screenshot 2023-11-02 at 8 24 03 AM" src="https://github.com/BenMKT/BlogApp/assets/44798044/05ee08bb-d346-4c4b-970d-1c8c2e047ad3">

Reply:
- Update tests to resolve bugs.
- For the next reviewer, please consider scheduling a quick zoom call in case of such recurrent errors to iron out together as when running tests in my terminal, they're all passing as per the screenshot below, and don't understand how the error is arising.

<img width="524" alt="Screenshot" src="https://github.com/BenMKT/BlogApp/assets/118020802/104874f7-4cd5-4be7-aed6-2ccc6389c675">